modules/Biosample/annotationProperty.csv

```diff
@@ -22,7 +22,7 @@ modelSystemName,"A group of presumed common ancestry with clear-cut physiologica
 nf2Genotype,"Genotype of NF2 gene in the biospecimen from which the data were derived, if known","-/-, +/-, +/+, +/?, -/?, Unknown",,FALSE,,neurofibromatosis,,,,nf2Genotype,Property,Biosample,annotationProperty,,Genotype,,
 nf1Genotype,"Genotype of NF1 gene in the biospecimen from which the data were derived, if known","-/-, +/-, +/+, +/?, -/?, Unknown",,FALSE,,neurofibromatosis,,,,nf1Genotype,Property,Biosample,annotationProperty,,Genotype,,
 specimenPreparationMethod,"Term that represents preservation of the sample before usage in, e.g. sequencing","Fresh collected, Flash frozen, FFPE, Cryopreserved, OCT, RNAlater, formalin-fixed, ethanol, Viably frozen",,FALSE,https://docs.gdc.cancer.gov/Data_Dictionary/viewer/#?view=table-definition-view&id=sample&anchor=preservation_method,ngs,,,,specimenPreparationMethod,Property,Biosample,annotationProperty,,Specimen_State,,
-age,A numeric value representing age of the individual. Use with ageUnit.,,ageUnit,FALSE,Sage Bionetworks,,,,,age,DataProperty,Biosample,annotationProperty,biosampleAnnotation,,,
+age,A numeric value representing age of the individual. Use with ageUnit.,,ageUnit,FALSE,Sage Bionetworks,,,,num,age,DataProperty,Biosample,annotationProperty,biosampleAnnotation,,,
```

Comment:
A couple of minor thoughts on this - some of our contributors (JHU Biobank i think?) don't report young children's exact ages, they do something like `<5`. 

Should we change this validation to regex that will permit the use of < or > (e.g. for `>89` for Safe Harbor deidentified data)?